schema/global/rooms.ts

```diff
@@ -172,3 +189,108 @@ export const RoomKindQuery = extendType({
 		t.crud.roomKinds({ filtering: true });
 	},
 });
+
+const roomFieldsType = {
+	name: stringArg(),
+	kind: stringArg(),
+	vol: intArg(),
+	vent: stringArg(),
+	units: list(UnitMutationType)
+};
+
+export const RoomStatus = mutationStatusType({
+	name: "RoomStatus",
+	definition(t) {
+		t.string('name', { description: `A string representation of the new Room's object identity; may be thereafter passed to e.g. \`updateRoom\``});
+	}
+});
+
+export const RoomMutations = extendType({
+	type: 'Mutation',
+	definition(t) {
+		t.nonNull.field('updateRoom', {
+			description: `Update room details.`,
+			args: {
```

Comment:
Would this work with just `args: roomType` (was : `args: roomFieldsType` before my own commits)?

Reply:
It works in only one line. I did the same also for unit.ts